lib/student_greeting_checker.rb

```diff
@@ -20,22 +21,23 @@ def check_all_ungreeted_students
   end
 
   def check(student, wiki)
-    Check.new(student, wiki, @greeters).update_greeting_status
+    Check.new(student, wiki, @greeters, @usernames).update_greeting_status
   end
 
   # Checks a single student to see if they are actually ungreeted, and updates
   # their greeted attribute if so.
   class Check
-    def initialize(student, wiki, greeters)
+    def initialize(student, wiki, greeters, usernames)
```

Comment:
Is `greeters` still used here? Perhaps `greeters` should become `greeter_usernames`? I suspect we only need one instance variable (passed along to the `Check` class) for the greeters for this whole file.

Reply:
updated!